### DISCUSSION TITLE
Définir "PAT" dans le diag

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1283] [Idées]
À la demande d'une utilisatrice, il semble judicieux de définir le sigle PAT dans le volet "Information des usagers et convives" du diagnostic car sa signification est ignorée de bien des utilisateurs :

<img width="799" alt="Capture d’écran 2022-04-12 à 14 07 34" src="https://user-images.githubusercontent.com/93709421/162959353-27ef3628-7f59-4b15-b0ab-5347b3fe60ca.png">

**Comment by alemangui** [comment]
Bien vu. Issue créé : https://github.com/betagouv/ma-cantine/issues/1385
